README.md

```diff
@@ -2,12 +2,19 @@
 
 # About the OpenSearch documentation repo
 
-The documentation repository contains the documentation for OpenSearch, the search, analytics, and visualization suite with advanced security, alerting, SQL support, automated index management, deep performance analysis, and more. You can find the rendered documentation at [opensearch.org/docs](https://opensearch.org/docs).
+The documentation repository contains the documentation for OpenSearch, the highly scalable and extensible open-source software suite for search, analytics, observability, and other data-intensive applications. You can find the rendered documentation at [opensearch.org/docs](https://opensearch.org/docs).
```

Comment:
Should this be the actual name of the repo ("documentation-website")? Do we think we need to include the added text describing OpenSearch, or would the reader likely already know what OpenSearch is if they're reading this file?